.github/workflows/release-python.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: Upload Python Package
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - release/*
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip build twine
+      - name: Build and publish
+        working-directory: ./python
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```

Comment:
@blaisep can you remove this file from main and resubmit it in a PR? 

We currently don't have anything to publish to PyPI. Once that is setup the PR to enable python releases can be made.